main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
```

Comment:
- Great setup of the user_input requests.
- A suggested refactoring would be to use error handling and user_input verification to prevent the code from crashing.
- Consider adding a condition as part of the while loop or inside the loop body:
-  One that will allow the current code to continue asking for the correct response when the input is invalid.
- Let me know if this helps!